src/Components/SingleArticle.jsx

```diff
@@ -8,27 +8,37 @@ import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
+import VoteButton from './VoteButton'
 
 export default function SingleArticle(){
 
   const { articleId } = useParams();
   const [article, setArticle] = useState(null);
   const [loading, setLoading] = useState(true);
   const [comments, setComments] = useState([]);
+  const [totalVotes, setTotalVotes] = useState(0);
 
   useEffect(() => {
     Promise.all([getArticleById(articleId), getCommentsById(articleId)])
```

Comment:
I would consider doing this separately as if either one of these promises rejects and errors, this will cause the entire promise to reject, even if one of the promises resolve. this means you wont be able to display the comments nor the articles if one of them fails to fetch. if you separate them, and e.g. only the comments fail, you will still be able to set and display the articles :)